src/System.Reflection.Metadata/tests/Metadata/Decoding/SignatureDecoderTests.cs

```diff
@@ -15,6 +15,19 @@ namespace System.Reflection.Metadata.Decoding.Tests
 {
     public partial class SignatureDecoderTests
     {
+        private static readonly string RuntimeAssemblyName =
+#if netstandard
```

Comment:
The define needs to be locally defined in the .csproj file until https://github.com/dotnet/corefx/pull/31470 comes through.